src/components/breadcrumb-item/daikin-breadcrumb-item.ts

```diff
@@ -83,13 +102,15 @@ export class DaikinBreadcrumbItem extends LitElement {
   target: string | null = null;
 
   /**
-   * Specify the link should show slash at the end or not
+   * Whether the link should have trailing slash.
```

Comment:
Sorry the slash is not in the link.

```suggestion
   * Whether the slash after the link should shown.
```